app/jobs/update_coffees_job.rb

```diff
@@ -0,0 +1,35 @@
+class UpdateCoffeesJob < ApplicationJob
+  queue_as :default
+
+  def perform(reference)
+    # TODO Catch error when scraper doesnt exist
+    scraper = Object.const_get "FetchCoffees::" + reference.classify
+
+    coffees = scraper.new.scrape
+    roaster = Roaster.find_by reference: reference
+
+    coffees.each do |coffee|
+      this_coffee = roaster.coffees.find_or_create_by!(url: coffee["url"]) do |new_coffee|
+        new_coffee.name = coffee["name"]
+        new_coffee.country = coffee["country"]
+        new_coffee.tasting_notes = coffee["tasting_notes"]
+      end
+
+      this_coffee.update(
+        country: coffee["country"],
+        tasting_notes: coffee["tasting_notes"],
+        region: coffee["region"],
+        town: coffee["town"],
+        lat: coffee["lat"],
+        lng: coffee["lng"],
+        process: coffee["process"],
+        altitude: coffee["altitude"],
+        variety: coffee["variety"],
+        producer: coffee["producer"],
+        available: coffee["available"],
+        description: coffee["description"]
+      )
```

Comment:
Could you do `this_coffee.update!(coffee)` instead of writing out all of the mappings?

Reply:
required a few extra changes, but yep, good call.